exercises/practice/palindrome-products/src/lib.rs

```diff
@@ -1,19 +1,15 @@
-/// `Palindrome` is a newtype which only exists when the contained value is a palindrome number in base ten.
-///
-/// A struct with a single field which is used to constrain behavior like this is called a "newtype", and its use is
-/// often referred to as the "newtype pattern". This is a fairly common pattern in Rust.
-#[derive(Debug, Clone, Copy, PartialEq, Eq)]
-pub struct Palindrome(u64);
+use std::collections::HashSet;
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct Palindrome;
```

Comment:
```suggestion
pub struct Palindrome {
    // TODO
}
```

While solving exercises, I sometimes wasn't sure what I was supposed to change and what to leave as it is. So I think it could remove some friction to mark this explicitly.

Reply:
Fair point, added.